recipes-kernel/linux/linux-fslc-imx_5.4.bbappend

```diff
@@ -0,0 +1,8 @@
+SRC_URI_imx7d-remarkable2 = "git://github.com/alistair23/linux.git;protocol=git;branch=${SRCBRANCH}"
+SRCBRANCH_imx7d-remarkable2 = "5.4-rM2-2.2.x-imx"
```

Comment:
I am totally fine you using the fork but I'd prefer you create a specific recipe (e.g. linux-remarkable2) instead of using a bbapped.

Reply:
Fixed!